ACadSharp/IO/DWG/DwgStreamReaders/DwgObjectReader.cs

```diff
@@ -1048,6 +1054,129 @@ private CadTemplate readUnlistedType(short classNumber)
 			return template;
 		}
 
+		#endregion
+
+		#region Evaluation Graph, Enhanced Block etc.
+
+		private CadTemplate readEvaluationGraph() {
+			EvaluationGraph evaluationGraph = new EvaluationGraph();
+			EvaluationGraphTemplate template = new EvaluationGraphTemplate(evaluationGraph);
+
+			this.readCommonNonEntityData(template);
+
+			//	DXF fields 96, 97 contain the value 5, here are three fields returning the same value 5
+			var val1 = _objectReader.ReadBitLong();
```

Comment:
see LibreDWG dwg.spec:
BL first_nodeid DXF 96
BL first_nodeid_copy DXF 97, always the same as first_nodeid
BL nodeCount